code/modules/clothing/masks/transmog.dm

```diff
@@ -143,7 +143,7 @@
 	..()
 	color = rgb(rand(0,255),rand(0,255),rand(0,255))
 	//Remove cockatrices because they're somewhat OP when player controlled
-	target_type = pick(existing_typesof(/mob/living/simple_animal) - (existing_typesof_list(blacklisted_mobs) + existing_typesof_list(boss_mobs)))
+	target_type = pick(existing_typesof(/mob/living/simple_animal) - (existing_typesof_list(blacklisted_mobs) + existing_typesof_list(boss_mobs) + /mob/living/simple_animal/scp_173 + existing_typesof(/mob/living/component)))
```

Comment:
?

Reply:
we don't want the mask to turn you into mobs that will move even when you're in control (especially in the case of component AI spiders that were never meant to be controlled)

Reply:
Essentially this is parity with #29627

Reply:
You shouldn't need to include this file at all since those changes were already made elsewhere

Reply:
But they were not? I meant parity as in "making transmog mask up to date with the other types we made illegal in the menagerie PR"

Reply:
Oh, I see then